src/components/createTestCase/CreateTestCase.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import { Button, HelperText, Label } from "@madie/madie-components";
+import { useLocation, useNavigate } from "react-router-dom";
+import { useFormik } from "formik";
+import tw from "twin.macro";
+import * as Yup from "yup";
+import TestCase from "../../models/TestCase";
+
+const FormControl = tw.div`mb-3`;
+const FormErrors = tw.div`h-6`;
+const TestCaseForm = tw.form`m-3`;
+const FormActions = tw.div`flex flex-row gap-2`;
+
+const TestCaseDescription = tw.textarea`
+  w-96
+  resize
+  h-24
+  rounded-md
+  sm:text-sm
+`;
+
+const CreateTestCase = () => {
+  const location = useLocation();
+  const navigate = useNavigate();
+  const [testCase, setTestCase] = useState<TestCase>();
+
+  const formik = useFormik({
+    initialValues: {
+      description: "",
+    } as TestCase,
+    validationSchema: Yup.object().shape({
+      description: Yup.string().required("Patient description is required"),
```

Comment:
don't think this is a required field.

Reply:
I'll remove the required validation in the next PR, in which I'll also be adding tests